paper.tex

```diff
@@ -63,6 +63,8 @@ \subsection{Generating models}
 
 \subsubsection{Image generating models}
 
+\textbf{Winston AI} is a commercial AI text detector that starts at 18\$ a month. They have two primary methods for checking if a text is AI written. Firstly Linguistic analysis analyses every text of its characteristics like repetitiveness. These characteristics are different between AI and humans. Secondly the other is Data Training, where the detector saw many AI written texts of one model and learned unique characteristics that this model uses which than can be detected.\cite{WinstonAI}
```

Comment:
```suggestion
\textbf{Winston AI} is a commercial AI text detector that starts at 18\$ a month. They have two primary methods for checking if a text is AI written. Firstly Linguistic analyse analyses every text for its characteristics like repetitiveness. These characteristics are different between AI and humans. Secondly the other is Data Training, where the detector saw many AI written texts of one model and learned unique characteristics that this model uses which than can be detected.\cite{WinstonAI}
```

Reply:
Is 'Linguistic analyse' a specific name ?
If not I would write it in lowercase

Reply:
'analyse' is not a noun at all, so in my opinion analysis is the right one here.

> Is 'Linguistic analysis' a specific name ?

Not really. The original source (see the bib-file) does once writes it uppercase, otherwise lowercase (in the middle of the sentence). I agree to @RaphaW and would orientate ourself on Wikipedia (https://en.wikipedia.org/wiki/Linguistics) and write it lowercase.